### PR TITLE
Update registry.yaml to add Weights & Biases examples

### PR DESCRIPTION
**Pull request by ash0ts**
## Summary

Update registry.yaml to add Weights & Biases examples for the cookbook website.

## Changes

Update registry.yaml

- Major Changes
  - Update registry.yaml to add Weights & Biases examples

## Motivation

Discoverability on the more popularly rendered website

## Self Review Checklist

- [ ] Is the writing easily skimmable?
  - Sections have informative titles.
  - Key takeaways are upfront.
  - Short paragraphs and topic sentences are used.
- [ ] Is the writing quality high?
  - Sentences are simple and unambiguous.
  - Demonstrative pronouns are avoided or clearly referenced.
  - No left-branching sentences.
- [ ] Is the content universally helpful?
  - Terminology is specific and avoids jargon.
  - Provides solutions to common problems.
  - Code examples are general and exportable.
- [ ] Is the content consistent?
  - Styling and formatting align with existing documentation.
  - Consistent use of punctuation and case.

When contributing a new example, make sure to add a new entry for it in [registry.yaml](/registry.yaml) to render it on the cookbook website.

**Note:** For additional guidelines on writing good documentation, check out [What Makes Documentation Good](https://cookbook.openai.com/what_makes_documentation_good).

### CODE REVIEW

**Comment by ash0ts** [comment]
@simonpfish I forgot to update the `registry.yaml`. Would appreciate a stamp

File: registry.yaml [hunk 1]
```diff
@@ -999,6 +999,24 @@
   path: examples/vector_databases/deeplake/deeplake_langchain_qa.ipynb
   date: 2023-09-30
   authors:
-    - FayazRahman
+    - ash0ts
```
Comment (simonpfish) [inline registry.yaml:1002]: was this intentional? 

```suggestion
    - FayazRahman
```

**Review by simonpfish — CHANGES_REQUESTED** [review]
Just one minor comment!

File: registry.yaml [hunk 2]
```diff
@@ -999,6 +999,24 @@
...
+- title: How to Finetune ChatGPT-3.5 and/or GPT-4 with Weights & Biases
+  path: examples/third_party_examples/GPT_finetuning_with_wandb.ipynb
+  date: 2023-10-04
+  authors:
+    - ash0ts
+  tags:
+    - tiktoken
+    - completions
+
+- title: OpenAI API Monitoring with W&B Weave
```
Comment (simonpfish) [inline registry.yaml:1015]: Let's maybe include the full name here for clarity at the top level. 

```suggestion
- title: OpenAI API Monitoring with Weights & Biases Weave
```

**Comment by ash0ts** [comment]
@simonpfish Added the suggestions you made. One nit, I called the `Finetuning example`: `Fine-tuning GPT with Weights & Biases`

**Comment by simonpfish** [comment]
Perfect! Merging.

**Pull request merged**